en/sql/datatype.rst

```diff
@@ -1103,7 +1103,7 @@ The following are the rules that are applied when using the character string typ
 
     Specify the number of a character string.
 
-    When the length of the character string entered exceeds the length specified, the excess characters are truncated.
+    When the length of the character string entered exceeds the length specified, the excess characters may be truncated on the insert/update operation if the allow_truncated_string configuration value is "yes" or error occurs if not.
```

Comment:
the excess characters may be truncated on the insert/update operation if the allow_truncated_string configuration value is "yes"or error occurs if not.
-->
1. the excess characters may be truncated in..... (on-->in)
2. if the allow_truncated_string configuration value is "yes" otherwise an error occurs. (or...not --> otherwise, added 'an')